frontend/src/common/analytics/events.ts

```diff
@@ -39,6 +39,7 @@ export enum EVENTS {
   WMG_FMG_ADD_GENES_CLICKED = "WMG_FMG_ADD_GENES_CLICKED",
   WMG_FMG_QUESTION_BUTTON_HOVER = "WMG_FMG_QUESTION_BUTTON_HOVER",
   WMG_FMG_DOCUMENTATION_CLICKED = "WMG_FMG_DOCUMENTATION_CLICKED",
+  WMG_TISSUE_EXPAND = "WMG_TISSUE_EXPAND",
```

Comment:
has this been added to Plausible? Thank you!

Reply:
Need to add!